ui/src/views/Tasks/TaskIndex/ListNamespaces/index.jsx

```diff
@@ -232,7 +233,11 @@ export default class ListNamespaces extends Component {
             </Fragment>
           )}
           {!loading && !hasNamespaces && !hasIndexedTasks && isSinglePath && (
-            <Typography variant="body2">No items for this page.</Typography>
+            <Typography variant="body2">
+              {searchTerm
+                ? `No items for this page with search term ${searchTerm}.`
+                : 'No items for this page.'}
+            </Typography>
```

Comment:
I think the reason why we had this hard-coded here was because we didn't support the `searchTerm` prop to the the table. Now that we do, we could probably just feed the search term to the table like the other views. Should we do that here?

Reply:
In the render method of this file, we use either `IndexNamespacesTable` or `IndexTaskNamespaceTable` to render the table. Are we able to provide the search term to those functions and let the table handle providing this message rather than hard-coding it here?